node-packages/commons/src/util/lagoon.ts

```diff
@@ -4,6 +4,9 @@ import { EnvKeyValue } from '../api';
 export const generateBuildId = () =>
   `lagoon-build-${Math.random().toString(36).substring(7)}`;
 
+export const variableOnlyBuild = () =>
```

Comment:
```suggestion
export const generateVariableOnlyBuildId = () =>
```
this would keep the naming of the closely related functions similar.